src/js/tech/html5.js

```diff
@@ -139,8 +137,6 @@ class Html5 extends Tech {
           el.appendChild(trackEl);
         }
       }
-
-      Lib.insertFirst(el, player.el());
```

Comment:
One thing we need to be aware of here is that the HTML5 tech only added it's element to the player if it wasn't reusing the embed code video tag that was [already in the player](https://github.com/videojs/video.js/blob/e1e69941a13d444525ccdf494d740eb1a3285320/src/js/player.js#L211). Now, the player will call insertFirst for the HTML5 tech element even if the tag is already there. I'm not sure what, if any impact that will have.